app/assets/stylesheets/_headers_from_future_juntos.scss

```diff
@@ -361,6 +384,39 @@
     color: rgba(241, 244, 244, 0.54);
   }
 }
+.btn-floating-action {
```

Comment:
Seria bacana a ordenação das propriedades por grupo, mais ou menos como nesse exemplo: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/